crates/beerus-rpc/src/api.rs

```diff
@@ -225,4 +225,11 @@ pub trait BeerusApi {
         request: FunctionCall,
         block_number: u64,
     ) -> Result<Vec<FieldElement>, Error>;
+
+    #[method(name = "getStorageAt")]
+    async fn get_storage_at(
+        &self,
+        contract_address: String,
```

Comment:
If I am not mistaken starknet_getStorageAt also takes a BlockId:
<img width="1259" alt="SCR-20230413-qbqi" src="https://user-images.githubusercontent.com/42912740/231823589-257f9730-07e9-4cac-b719-f7c174e56e79.png">

As seen here.

Reply:
I agree i will create an issue and take care of the get_nonce now.

Reply:
Will create issues for adding blockId as a parameter after merging BlockId PR: https://github.com/keep-starknet-strange/beerus/pull/391

Well done.